src/components/badge/badge_group.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { HTMLAttributes, Ref } from 'react';
+import classNames from 'classnames';
+import { CommonProps, keysOf } from '../common';
+
+export type BadgeGroupGutterSize = keyof typeof gutterSizeToClassNameMap;
```

Comment:
This isn't used anywhere else and so doesn't need to get exported. I would also make sure this comes **after** the declaration of `gutterSizeToClassNameMap`

```suggestion
type BadgeGroupGutterSize = keyof typeof gutterSizeToClassNameMap;
```